cephfs/cephfs.go

```diff
@@ -84,6 +118,55 @@ func (mount *MountInfo) MakeDir(path string, mode uint32) error {
 	if ret == 0 {
 		return nil
 	} else {
+		log.Errorf("MakeDir: Failed to make directory %s", path)
 		return CephError(ret)
 	}
 }
+
+func (mount *MountInfo) RemoveDir(path string) error {
+	c_path := C.CString(path)
+	defer C.free(unsafe.Pointer(c_path))
+
+	ret := C.ceph_rmdir(mount.mount, c_path)
+	if ret == 0 {
+		return nil
+	} else {
+		log.Errorf("RemoveDir: Failed to remove directory")
+		return CephError(ret)
+	}
+}
+
+func (mount *MountInfo) Chmod(path string, mode uint32) error {
+	c_path := C.CString(path)
+	defer C.free(unsafe.Pointer(c_path))
+
+	ret := C.ceph_chmod(mount.mount, c_path, C.mode_t(mode))
+	if ret == 0 {
+		return nil
+	} else {
+		log.Errorf("Chmod: Failed to chmod :%s", path)
+		return CephError(ret)
+	}
+}
+
+func (mount *MountInfo) Chown(path string, user uint32, group uint32) error {
+	c_path := C.CString(path)
+	defer C.free(unsafe.Pointer(c_path))
+
+	ret := C.ceph_chown(mount.mount, c_path, C.int(user), C.int(group))
+	if ret == 0 {
```

Comment:
use guarde clause instead of creating if/else condition...

Reply:
Addressed: https://github.com/ceph/go-ceph/pull/74/commits/d23a19afe9b5e944b25c16c9268a7a3dffefb438